### PR TITLE
Update NPC locations according client's navi file

### PR DESCRIPTION
**Pull request by zackdreaver**
* **Addressed Issue(s)**: n/a

* **Server Mode**: Both

* **Description of Pull Request**: Fixed some NPC locations according client's navi file

### CODE REVIEW

File: npc/pre-re/cities/prontera.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//= 1.7 Removed Duplicates [Silent]
+//= 1.8 Moved Culvert to quests/quests_prontera.txt
+//= 1.9 Rescripted to Aegis 10.3 standards. [L0ne_W0lf]
+//= 2.0 Removed "Hans Solo", as he is a Jawaii NPC. [L0ne_W0lf]
+//= 2.1 Disabled the Library Curator (bugreport:4105) [L0ne_W0lf]
+//============================================================
+
+// Prontera
+//============================================================
+prontera,160,330,4	duplicate(prtguard)	Guard#5pront	105
```
Comment (Lemongrass3110) [inline npc/pre-re/cities/prontera.txt]: Missing newline

File: npc/pre-re/cities/prontera.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+//= Spellchecked [massdriller]
+//= 1.4 Optimized. Novices > 20 BaseJob can go to Culvert, too [Lupus]
+//= 1.5a Added a Jawaii related npc [MasterOfMuppets]
+//= 1.6 Replaced sewer_prt with (MISC_QUEST & 8) [Lupus]
+//= 1.7 Removed Duplicates [Silent]
+//= 1.8 Moved Culvert to quests/quests_prontera.txt
+//= 1.9 Rescripted to Aegis 10.3 standards. [L0ne_W0lf]
+//= 2.0 Removed "Hans Solo", as he is a Jawaii NPC. [L0ne_W0lf]
+//= 2.1 Disabled the Library Curator (bugreport:4105) [L0ne_W0lf]
+//============================================================
```
Comment (aleos89) [inline npc/pre-re/cities/prontera.txt]: Do you mind using the newer header format? You can see an example in `npc/re/other/achievements.txt`. Thanks!

Comment (zackdreaver) [reply npc/pre-re/cities/prontera.txt]: Sure

**Comment by zackdreaver** [comment]
I've updated \npc\re\guides file header as well,
I can push it if you want.

**Comment by aleos89** [comment]
It might be best to eventually create a separate PR which updates the remaining NPC to the new header format.

**Comment by zackdreaver** [comment]
I have no more things to add, if there something that someone want to add, feel free to request or push it.

**Pull request merged**